tox.ini

```diff
@@ -107,8 +107,7 @@ set_env =
 #  constrain_package_deps = "false"
 ##
 deps =
-    zope.testrunner
-    -c https://dist.plone.org/release/6.0-dev/constraints.txt
+    zope.testrunner -c https://dist.plone.org/release/6.0-dev/constraints.txt
```

Comment:
@rohnsha0 hmm, it looks like this hasn't been fixed in `meta` 2.0.0 yet. But there were changes made to the `2.x` branch that aren't in a release yet. Maybe those changes will fix this bug. You can try cloning the `meta` repository, and installing from the clone:

```
venv/bin/pip install -e .
```

Reply:
done!

Reply:
@rohnsha0 I investigated the issue and this is not a `plone/meta` bug. See: https://github.com/plone/plone.scale/issues/105#issuecomment-2689085987

Reply:
@rohnsha0 Since using the 2.x branch of `plone/meta` didn't solve the problem, for now I think it's better to use relase 2.0.0.

Reply:
since the constraint file was pinning `webtest` twice. as per the [comment](https://github.com/plone/plone.scale/issues/105#issuecomment-2689250367), the version issue was fixed and so are the the jobs (ofc after re-running it). its all green now!